### PR TITLE
Suggest fixes

### PR DESCRIPTION
**Pull request by vogre**
For https://github.com/reshufflehq/reshuffle/issues/204

### CODE REVIEW

File: test/hooks/not-found.test.ts [hunk 1]
```diff
@@ -1,36 +1,14 @@
...
-    })
-
-  test
-    .stderr()
-    .hook('command_not_found', {id: 'commans'})
-    .catch('Run @oclif/plugin-not-found help for a list of available commands.')
-    .end('runs hook with not found error after no input timeout', (ctx: any) => {
-      expect(ctx.stderr).to.be.contain('Warning: commans is not a @oclif/plugin-not-found command.\n')
-    })
+    .catch('commans is not a @binaris/oclif-plugin-not-found command.\n' +
```
Comment (arielshaqed) [inline test/hooks/not-found.test.ts:10]:
```suggestion
    .catch('command is not a @binaris/oclif-plugin-not-found command.\n' +
```
Probably also want to fix `id` on the line above

**Review by arielshaqed — APPROVED** [review]
This is a behavioural change.  I think it should be user-selectable, e.g. via an environment variable.

Approving as (I believe) this is immediately useful to us.

Thanks!

File: test/hooks/not-found.test.ts (hunk 1 above)
Comment (vogre) [reply test/hooks/not-found.test.ts:10]: This confused me at first as well. The typo of `commands` -> `commans` is intentional, would a different example `run` -> `rn` be better?

Comment (arielshaqed) [reply test/hooks/not-found.test.ts:10]: Ouch, no, it just _is_ confusing.  Either add a comment or just leave it alone.

**Review by arielshaqed — APPROVED** [review]
Thanks!

**Pull request merged**